jaeger-core/src/test/java/io/jaegertracing/internal/JaegerSpanTest.java

```diff
@@ -485,4 +493,34 @@ public void testAsChildOfAcceptNull() {
     jaegerSpan.finish();
     assertTrue(jaegerSpan.getReferences().isEmpty());
   }
+
+  @Test
+  public void concurrentConcurrentLogs() {
+    jaegerSpan.getTags();
+    JaegerTracer tracer = new JaegerTracer.Builder("foo")
+        .withReporter(this.reporter)
+        .withSampler(new ConstSampler(true))
+        .build();
+
+    JaegerSpan span = tracer.buildSpan("foo").start();
+    span.log("foo");
+    for (LogData log: span.getLogs()) {
+      span.log("foo2");
```

Comment:
Is this test failing when ran against the current master?

Reply:
I think so